### ISSUE TITLE
Great module. How about a release?

### ISSUE THREAD

**Issue by yorkshire-pudding** [backdrop-contrib/views_autocomplete_filters#2]
Hi @jenlampton and @larsdesigns 

This is just the module I was looking for.  I've installed and it appears to work well.
Is it worth doing a release, even if a beta?

**Comment by jenlampton** [comment]
Yep, we can do a release!

**Comment by larsdesigns** [comment]
Yes, sounds good to me.On Jan 17, 2024, at 6:31 AM, Martin Price ***@***.***> wrote:﻿
Hi @jenlampton and @larsdesigns
This is just the module I was looking for.  I've installed and it appears to work well.
Is it worth doing a release, even if a beta?

**Comment by yorkshire-pudding** [comment]
Bump @jenlampton @larsdesigns

**Comment by sudipto68** [comment]
@jenlampton I've installed and it appears to work well for me. Can we please have a release for this module?

**Comment by yorkshire-pudding** [comment]
Hi @jenlampton and @larsdesigns 
I've just suggested to a forum user that they might want this module.  Any chance it can be released in the near future?
